.evergreen/is_python3.py

```diff
@@ -0,0 +1,45 @@
+# Determine whether the current Python executable is usable for Drivers Evergreen Tools.
+# Exit status will determine compatibility.
+
+import sys
+
+MIN_PYTHON = "3.9"
```

Comment:
```suggestion
REQUIRED_PYTHON_VERSION_MAJOR = 3
REQUIRED_PYTHON_VERSION_MINOR = 9
```

Might be easier to maintain (by avoiding string split operations). e.g. further below:

```python
if not (sys.version_info[0] == REQUIRED_PYTHON_VERSION_MAJOR and sys.version_info[1] >= REQUIRED_PYTHON_VERSION_MINOR):  # noqa: YTT201
    error_out(f"Unsupported version of python")
```

There is atm no output of the full required version, but if needed, it could use an f-string.